### ISSUE TITLE
Week 2 Project: New York Times Article Search

### ISSUE THREAD

**Issue by allo86** [allo86/NYTimes-Search#1]
My app is complete. Please review. /cc @codepath @codepathreview

**Comment by codepathreview** [comment]
:+1: Great work again, following up on last week's assignment. A few notes after checking out the assignment:
- **Loved the usage of FAB for filters/share**
- Properly reused the single base method in search activity to fetch results for both initial load and the pagination
- Nice to see you added the search box to the ActionBar!
- Properly used spinners & check-boxes (with populated values) to allow users to select appropriate values.
- Nice work setting up shared preferences to persist the settings between activities
- Good to see you used a filters object  to represent the filters to apply to the query
-  Nice job adding the share intent to the article detail view!
- Consider adding [progress bars](http://guides.codepath.com/android/Handling-ProgressBars#progress-within-actionbar) to your app whenever a network request goes out

Also, here's a more detailed [Project 2 Feedback Guide here](http://courses.codepath.com/snippets/intro_to_android/project_2_feedback) which covers the most common issues with this submitted project. Read through the [feedback guide point-by-point](http://courses.codepath.com/snippets/intro_to_android/project_2_feedback) to determine how you could improve your submission.

Let us know if you have any other thoughts or questions about this assignment. The next assignment (Twitter Client) will be especially important since it introduces the majority of the remaining pieces necessary to build a fully functional API client with complex feeds of data and user creation.

**Comment by allo86** [comment]
Thanks @codepathreview 

About your notes, actually I am using a progress bar in the Toolbar for the Search Activity. It is with the "small" style and maybe I should have used the bigger one, to make it more visible. I also used a progress bar, with horizontal style, in the Detail Activity, to show the web's load progress. Was it the right approach?

**Comment by codepathreview** [comment]
Yup, just noticed. Generally, use the progress bar in the place where content is going to be shown and then toggle its visibility once the network response is returned (either success or failure).

Looks great though.

**Comment by allo86** [comment]
Thanks @codepathreview
